Mid.css

```diff
@@ -0,0 +1,214 @@
+body
+{ 
```

Comment:
keep this { inline with selector

body {
    // Your Code
}

Reply:
Done